Sources/Rectangle.swift

```diff
@@ -19,7 +20,16 @@ public class Rectangle : Shape {
         width = width * scaleFactor
     }
     
-    public func getArea() -> Int {
-        return height * width
+    public func getArea() -> Float {
```

Comment:
If we want to have non-integer areas, we need to allow non-integer dimensions when creating the rectangle or setting the area.